collect_app/src/main/res/layout/form_entry_end.xml

```diff
@@ -112,7 +114,9 @@ the specific language governing permissions and limitations under the License.
                 app:layout_constraintBottom_toBottomOf="parent"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toEndOf="@id/save_as_draft"
-                app:layout_constraintTop_toTopOf="parent" />
+                app:layout_constraintTop_toTopOf="parent"
+                app:icon="@drawable/ic_send_24"
+                app:iconGravity="textStart" />
```

Comment:
I think we do this for all Material buttons. Should we set our own `?materialButtonOutlinedIconStyle` and `?materialButtonIconStyle` that extend the default styles and add this do you think?

Reply:
Not for all, `first_launch_layout` is one example. There are some more but in other cases, we usually use `wrap_content` as `android:layout_width`, and then that difference is not visible. @alyblenkin what do you think should we always remove that space between the icon and the text? If so please file a separate issue.